src/seedu/addressbook/data/person/Address.java

```diff
@@ -12,11 +12,17 @@
 	public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses need to be in the form BLOCK, STREET, UNIT, POSTAL_CODE";
 	public static final String ADDRESS_VALIDATION_REGEX = ".+?, .+?, .+?, .+";
 
-	public final Block block;
-	public final Street street;
-	public final Unit unit;
-	public final PostalCode postalCode;
+	private final Block block;
+	private final Street street;
+	private final Unit unit;
+	private final PostalCode postalCode;
```

Comment:
Good job on remembering to encapsulate the class attributes, but try to do it in a separate commit next time.